AutoRest/Generators/Ruby/Ruby.Tests/RspecTests/validation_spec.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+
+$: << 'RspecTests/Generated/validation'
+
+require 'rspec'
+require 'validation'
+
+describe 'Validation Module' do
+
+  before(:all) do
+    @base_url = ENV['StubServerURI']
+
+    dummyToken = 'dummy12321343423'
+    @credentials = MsRest::TokenCredentials.new(dummyToken)
+
+    @client = ValidationModule::AutoRestValidationTest.new(@credentials, @base_url)
+  end
+
+  it 'should works with constant in path' do

```

Comment:
corrected

---
In reply to: [69042367](https://github.com/Azure/autorest/pull/1188#discussion_r69042367) [](ancestors = 69042367)